feat(demo): implement emsco_display for content types

Davidmattei opened the pull request:
| Q              | A |
|----------------|---|
| Bug fix?       |  n |
| New feature?   |  y |
| BC breaks?     |  n |
| Deprecations?  | n  |
| Fixed tickets? |  n |
| Documentation? | n  |

Davidmattei committed:
fix: legacy search should not redirect to default overview

Davidmattei committed:
feat: add display form field for news and pages

Davidmattei committed:
fix: PostProcessingService unset null values

If the objectArray contains a property with null value, isset is returning false. Array_key_exists is better.

Davidmattei committed:
feat: add overview and display for feature, release and user_group

Davidmattei committed:
feat(demo): version field use version analyzer

Correctly sort overviews

Davidmattei committed:
fix: backup all configs from demo (__fields__ removed)

theus77 commented on demo/configs/admin/content-type/release.json:
```diff
@@ -66,7 +66,7 @@
                                 "mappingOptions": {
                                     "index": null,
-                                    "analyzer": "keyword",
+                                    "analyzer": "version",
```
Nice fix: it was not possible on "old" elasticsearch ..... introduce in around 7.10 (but missing in 7.6.0)

Davidmattei replied:
Applied see next 2 commits, for now I will already merge :)

theus77 reviewed (approved):
THe here is some code in the templates to sort the versions with a |sort twig filter. It can be replace by a sort in the elasticsearch query now

Davidmattei committed:
fix: remark remove version sorting

Elasticsearch can now sort the releases

Davidmattei committed:
refactor: correctly sort releases and features

Features are sorted by category title (nl/fr) and then on feature title (nl/fr).
